### ISSUE TITLE
Skeleton components

### ISSUE THREAD

**Issue by MitchellNZ** [Megabit/Blazorise#1086]
**Loading "Skeleton" feature.**

New component to provide a generic loading templates.
Similar to this [Ant Design implentation](https://ant.design/components/skeleton/).

This is provides a visually appealing way for the user to wait on data loading from the server.

Example:
![image](https://user-images.githubusercontent.com/6387861/86543439-95493000-bf72-11ea-858b-94745e5bf001.png)

**Comment by MitchellNZ** [comment]
@stsrki this is something I plan to work on when I'm finished with Bar/Layout improvements!

**Comment by stsrki** [comment]
I remember we already talked about this one. Definitely would be good to have.

**Comment by stsrki** [comment]
@MitchellNZ Going on the backlog. Just reopen when you're ready to start!

**Comment by StephenOTT** [comment]
Would be great to have this.  Would be great to remove a lot of the typical `@if(myobject== null){}else{}`

**Comment by StephenOTT** [comment]
Typical types of recurring config:

1. Loading template
2. timeout config
3. error template
4. simple fade transition

**Comment by enisn** [comment]
> Would be great to have this. Would be great to remove a lot of the typical `@if(myobject== null){}else{}`

Yeah, I agree with that. 
If there can be an easier way without typing if-else everywhere might be useful.
Like that XF library:
https://github.com/HorusSoftwareUY/Xamarin.Forms.Skeleton
It makes it over existing components, you can override only when you want to customize. If something like that can be done with blazorise components, it'll be awesome 🚀

**Comment by stsrki** [comment]
It will come in time. The component is already scheduled for the next version, v1.1.

**Comment by stsrki** [comment]
https://steven-giesel.com/blogPost/d1884385-90de-43a1-ae24-0178495f1484

**Comment by XanderStoffels** [comment]
Bump! Any updates on this component?
Was looking for a skeleton component today, so I stumbled upon this issue. It would be a nice to have!

**Comment by stsrki** [comment]
I think we can try it for the next release now that the team is a little bigger.

**Comment by stsrki** [comment]
- Skeleton
  - SkeletonImage
  - SkeletonParagraph


```razor
<Skeleton>
  <SkeletonParagraph />
</Skeleton>
```
